content/news/playtest-20150524.md

```diff
@@ -0,0 +1,55 @@
+It has been a month of busy development. Time to get back to testing.
+
+The mayor changes this time around include:
+
+  - A whole new in-game map editor with all the benefits that come from it.
+  - Improved system integration with `openra://` server join URLs and instant `.orarep` playback.
+  - The Dune 2000 mod got building upgrades which re-creates the tech tree of the original.
```

Comment:
The english here is a bit off.  How about something like "Implemented the original Dune 2000 tech tree, with production building upgrades that unlock additional technology"?